db/helpers/common.js

```diff
@@ -47,7 +47,8 @@ function augmentSampleWithSubjectInfo(seq, inst) {
       inst.dataValues.subject = sub;
 
       // adding absolutePath to sample instance
-      inst.dataValues.absolutePath = sub.absolutePath;
+      if (sub)
```

Comment:
please remember to run gulp style before submitting pr

we enclose all ifs--even single-line blocks--in curly braces